main.js

```diff
@@ -1,17 +1,59 @@
-// VARIABLE AND ELEMENTS SETUP
-const books = [];
-const booksWrapper = document.getElementById('book-collection-wrapper');
 const addButton = document.getElementById('add_button');
-let deleteBook = function () {};
+const booksWrapper = document.getElementById('book-collection-wrapper');
+let books = [];
+let populateBookSection = () => {};
+class Books {
+  constructor() {
+    this.title = '';
+    this.author = '';
+    this.booksArr = books;
+  }
+
+  deleteBook(event) {
+    const local = JSON.parse(localStorage.getItem('books'));
+    const bookId = event.target.id;
+    if (local === null) {
+      this.booksArr.length = 0;
+    } else {
+      this.booksArr.splice(bookId, 1);
+      books = this.booksArr;
+    }
+    // eslint-disable-next-line
```

Comment:
Awesome work on your project! You've done a fantastic job, and everything is looking good. I just wanted to bring up one small thing that might need your attention:
- [ ] If you don't mind, could you please consider keeping the linter rules enabled? By sticking to these rules, your project will have an even more polished and professional feel. If you could apply this suggestion to the instances where you've disabled eslint in your code, that would be fantastic.